### PR TITLE
feat: add "Adding Pages" card to the home page

### PR DESCRIPTION
**Pull request by PeterShershov**
Notice that this card does not link to a page, instead it opens an overlay within itself that explains the task,

image

### CODE REVIEW

File: app/routes/_index/pages-overlay.module.scss [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+  color: black;
+  padding: 24px;
+  margin: 0px 32px;
+  gap: 16px;
+  text-wrap: pretty;
+
+  h1 {
+    font: $card-title;
+    color: $black;
+  }
```
Comment (matangeorgi) [inline app/routes/_index/pages-overlay.module.scss:17]: is there a reason its not another selector for the title?

File: app/routes/_index/pages-overlay.tsx [hunk 2]
```diff
@@ -0,0 +1,20 @@
+import React from 'react';
```
Comment (matangeorgi) [inline app/routes/_index/pages-overlay.tsx:1]: not needed

**Review by matangeorgi — APPROVED** [review]

File: app/routes/_index/pages-overlay.module.scss (hunk 1 above)
Comment (PeterShershov) [reply app/routes/_index/pages-overlay.module.scss:17]: Just minimizing the noise in the jsx by not adding a class.

**Comment by PeterShershov** [comment]
@omerken we need a better icon

image

**Pull request merged**